Public/Src/Cache/ContentStore/App/Application.cs

```diff
@@ -626,7 +626,7 @@ private void VerifyCachePathOrNameProvided(string name, string path)
 
             var config = new DistributedCacheServiceConfiguration(localCasSettings, dcs);
 
-            return new DistributedCacheServiceArguments(_logger, copier, pathTransformer, copyRequester, distributedCacheServiceHost, host, ct, dataRootPath, config, null);
+            return new DistributedCacheServiceArguments(_logger, copier, pathTransformer, copyRequester, distributedCacheServiceHost, host, ct, dataRootPath, config, null, true);

```

Comment:
Where should the config be located?

Reply:
I would put it on DistributedContentSettings for now.

---
In reply to: [314883546](https://github.com/microsoft/BuildXL/pull/751#discussion_r314883546) [](ancestors = 314883546)